docs/data/Tuple/instance/toJSON.md

```diff
@@ -0,0 +1,16 @@
+# `Tuple#toJSON()`
+
+Converts the `instance` to a [`JSON`][JSON] representation.
+
+## Returns
+
+* `(String)`: The `instance` as a [`JSON`][JSON] formatted `String`.
+
+```javascript
+const tuple = Tuple.of("a", 1);
+
+tuple.toJSON();
+// => '{"tuple":[1, "a"]}'
```

Comment:
Why this format instead of `{"first": 1, "second": "a"}`?

Reply:
I wanted a means to declare the type of the thing as well as the thing itself, but not focus too much on the names of the underlying properties. I had considered what you had there as well as:

```javascript
{
  "couple": [1, "a"]
}
{
  "tuple": {
    "size": 2,
    "values": [1, "a"]
  }
}
```

And some others I cannot recall. The key thing with `Tuple` is that it is an _ordered_ sequence. This is why I chose an `Array` for the JSONified version, but I don't like the mixed typing. I am open to suggestions.

Reply:
If we have no plans to support anything beyond 2 items, I think the `size` field is redundant.  I don't have a huge emotional stake in this, but the format you documented just seemed different to what I had expected.